services/brig/src/Brig/User/Auth.hs

```diff
@@ -67,6 +67,7 @@ sendLoginCode phone call force = do
     case user of
         Nothing -> throwE $ SendLoginInvalidPhone phone
         Just  u -> do
+            Log.debug $ field "user" (toByteString u) . msg (val "User.sendLoginCode")
```

Comment:
I think this is not necessary given it's already covered by `login` ?

Reply:
No this is part of the `/login/send` endpoint instead of the `/login` endpoint if I understood correctly